lib/hets.rb

```diff
@@ -107,6 +107,15 @@ def self.parse_via_api(resource, url_catalog = [], structure_only: false)
     parse_caller.call(iri, with_mode: mode)
   end
 
+  def self.prove_via_api(resource, url_catalog = [])
+    options = {}
+    if resource.in_distributed?
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.